environment.yml

```diff
@@ -6,28 +6,30 @@ dependencies:
   - python=3.11
   - pip
   - numpy
-  - astropy-base=5
+  - astropy-base
   - black
   - nbsphinx
-  - ctapipe=0.19.2
-  - gammapy=1.1
-  - ctaplot~=0.6.4
+  - ctapipe>=0.25.0
```

Comment:
this will take also ctapipe `v0.26.0`. Is it intended? One needs to check if ctapipe `v0.26.0` is compatible with these changes.